recipes/azure-common/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.1.4" %}
+
+package:
+  name: azure-common
+  version: {{ version }}
+
+source:
+  fn: azure-common-{{ version }}.zip
+  url: https://pypi.io/packages/source/a/azure-common/azure-common-{{ version }}.zip
+  sha256: f8c8d97d0a7de202a47d7081c39c0e4a827c78900719d02c2ebe936e44ff152f
+  
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - azure-nspkg
+  run:
+    - python
+    - azure-nspkg
+
+test:
+  imports:
+    - azure
+    - azure.common
+  commands:
+    - python -m unittest discover -p "test_*.py"
+
+about:
+  home: https://github.com/Azure/azure-sdk-for-python
+  license: Apache License 2.0
```

Comment:
I won't comment on all of these. Suffice it to say this would be a welcome change for the other recipes too.